pyne/r2s.py

```diff
@@ -11,8 +11,65 @@
 warn(__name__ + " is not yet QA compliant.", QAWarning)
 
 
-def irradiation_setup(flux_mesh, cell_mats, alara_params, tally_num=4,
-                      geom=None, num_rays=10, grid=False, flux_tag="n_flux",
+def resolve_mesh(mesh_reference, tally_num = None, flux_tag = "n_flux", output_material=False):
+
```

Comment:
I don't think a blank line is advisable here. For the docstring to be read properly by many autodoc tools, it really needs to be right after the def line.

```suggestion
```